### PR TITLE
Result: rename unwrap() to get()

### PR DESCRIPTION
**Pull request by msgilligan**
I borrowed `unwrap()` from Rust. `get()` is more consistent with standard Java classes.  (And 3 fewer characters 🤓)

### CODE REVIEW

File: secp256k1-examples-java/src/main/java/org/bitcoinj/secp256k1/examples/Schnorr.java [hunk 1]
```diff
@@ -18,10 +18,8 @@
 import org.bitcoinj.secp256k1.api.P256K1KeyPair;
 import org.bitcoinj.secp256k1.api.P256K1XOnlyPubKey;
 import org.bitcoinj.secp256k1.api.P256k1PubKey;
-import org.bitcoinj.secp256k1.bouncy.BouncyPrivKey;
```
Comment (schildbach) [inline secp256k1-examples-java/src/main/java/org/bitcoinj/secp256k1/examples/Schnorr.java:21]: Unrelated import changes.

Comment (msgilligan) [reply secp256k1-examples-java/src/main/java/org/bitcoinj/secp256k1/examples/Schnorr.java]: Fixed and force-pushed.

File: secp256k1-examples-java/src/main/java/org/bitcoinj/secp256k1/examples/Schnorr.java [hunk 2]
```diff
@@ -18,10 +18,8 @@
...
 import org.bitcoinj.secp256k1.foreign.Secp256k1Foreign;
 
-import java.math.BigInteger;
```
[the remark this answers is not in the document]
Comment (msgilligan) [reply secp256k1-examples-java/src/main/java/org/bitcoinj/secp256k1/examples/Schnorr.java]: Fixed and force-pushed.

**Review by schildbach — APPROVED** [review]
